src/stores/storage.js

```diff
@@ -0,0 +1,29 @@
+import { getDownloadURL, ref, uploadBytes } from 'firebase/storage'
+import { defineStore } from 'pinia'
+import { storage } from 'src/firebase'
+
+export const useStorageStore = defineStore('storage', {
+  actions: {
+    async uploadArtistPhoto(file, filename) {
+      const storageRef = ref(storage, `images/prompt-${filename}-artist`)
+
+      await uploadBytes(storageRef, file)
+
+      return getDownloadURL(storageRef).then((url) => url)
+    },
+
+    async uploadArts(files, filename) {
+      const arts = []
```

Comment:
This may be the source of the empty array uploading?

Reply:
Yes, this could be one of the causes of the problem.
I'm looking into it.